tests/snappi_tests/utils/metrics.py

```diff
@@ -0,0 +1,116 @@
+"""
+This file defines the classes receiving metrics from snappi tests and processing them.
+"""
+import logging
+import json
+import datetime
+import time
+
+from copy import deepcopy
+from pprint import pprint
+from typing import Dict, Final, List, Union
+
+# Only certain labels are allowed
+METRIC_LABEL_TEST_TESTBED: Final[str] = "test.testbed"
+METRIC_LABEL_TEST_BUILD: Final[str] = "test.os.version"
```

Comment:
this should be device not test.os.version

Reply:
This label is not for device, it is for the running build version. We used the standard name listed in resource.ex.

Reply:
I am a bit confused...

the test build is referring to the build that we are going to install to all DUTs before the nightly test starts (so we can test the nightly builds), so this is essentially the OS version that we get from the DUTs.

from definition "standard" point of view, the "os.version" is a standard name that being defined in OTEL, while "test.os.version" doesn't exist.
- https://opentelemetry.io/docs/specs/semconv/resource/
- https://docs.rs/opentelemetry-semantic-conventions/latest/opentelemetry_semantic_conventions/attribute/index.html

not sure this clarifies or not.